src/validate.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable quotes */
+
+export const check = (todo, id) => {
+  const findTodo = todo.find((item) => id === item.index);
+  findTodo.completed = !findTodo.completed;
+  localStorage.setItem('todo', JSON.stringify(todo));
+};
+
+export const clearCompleted = (todo) => {
+  const filterTodo = todo.filter((item) => {
+    if (item.completed !== true) {
+      return item;
+    }
+    return false;
+  });
```

Comment:
I think you are you can simply rewrite the function as a one-liner! Something like
```
const filterTodo = todo.filter((item) => item.completed === false);
```

Reply:
Thanks for the feedback, I will implement the changes😃😃😃